Example/CellDataCoupler/BaseTableViewCell.swift

```diff
@@ -12,32 +12,31 @@ import CellDataCoupler
 class BaseTableViewCell<T>: ReusableCell<T> {
     var selectionColor: UIColor?
     
-    required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
+    private func getSelectionColor() -> UIColor? {
```

Comment:
I see the rationale behind this, but my concern is that the developers can no longer find the selection color when the default is used. It looks like the selection color would be nil, but in reality it might be blue. This is misleading.

Reply:
A fair point!  I'll refactor.

Reply:
The thing is, `selectionStyle` isn't reliable.

Normally, we have to set `selectedBackgroundView` with an empty UIView, and set that view's background color to whatever selection color we want.  `selectionStyle` still isn't accurate in this scenario either.

But I see your point, I'll do a quick refactor, let me know what you think.

Reply:
I'll rollback the change for now (selection will remain broken), and we'll discuss this at the next team meeting.